definitions.h

```diff
@@ -221,8 +223,32 @@ class BufferImage : public Buffer2D<PIXEL>
  * designed/implemented by the programmer. 
  **************************************************/
 class Attributes
-{      
+{     
+    private:
+        double u;
+        double v;
+        double r;
+        double g;
+        double b; 
```

Comment:
To make this class more flexible you could have declared a vector:

std::vector<double> vAttr;